platform/ios/src/MGLMapView.mm

```diff
@@ -1926,6 +1926,19 @@ - (MGLCoordinateBounds)convertRect:(CGRect)rect toCoordinateBoundsFromView:(null
     return MGLCoordinateBoundsFromLatLngBounds([self convertRect:rect toLatLngBoundsFromView:view]);
 }
 
+- (CGRect)convertCoordinateBounds:(MGLCoordinateBounds)bounds toRectToView:(nullable UIView *)view
+{
+    return [self convertLatLngBounds:MGLLatLngBoundsFromCoordinateBounds(bounds) toRectToView:view];
+}
+
+/// Converts a geographic bounding box to a rectangle in the view’s coordinate
```

Comment:
Heh, not really. I started doing that in the OS X SDK for short doc comments in implementation files. Xcode’s Quick Help feature is the only consumer of these comments, and it supports the `///` syntax. (For fun, check out the local variable doc comments I added to [`flyTo()`](https://github.com/mapbox/mapbox-gl-native/blob/b6fca670896e6351a03916fd9fb646e1970d3ebe/src/mbgl/map/transform.cpp#L382-L389) – Quick Help picks those up too.)